packages/storybook-ui-components/stories/Badge.stories.tsx

```diff
@@ -29,11 +30,22 @@ const Label: FunctionComponent<{
 );
 
 export const Example = () => (
-  <section>
+  <StoryContainer>
+
     <h1>Badges</h1>
 
+    <StoryDescription>Use badges to visually label UI objects for quick recognition, status indication or navigation. Badges can be all-caps or lowercase. There are 5 badge intents:
+
+      <ul>
+      <li> <strong>Neutral:</strong> Use to indicate a default state </li>
+      <li> <strong>Success:</strong> Use to indicate a successful state </li>
+      <li> <strong>Warning:</strong>  Use to indicate that there is something that requires attention </li>
+      <li> <strong>Danger:</strong> Use to indicate a failed state. </li>
+      </ul>
+    </StoryDescription>
+
     <section>
-      <h3>Intents</h3>
+      <h4>Intents</h4>
```

Comment:
Again, not a change to make here but it seems like with the addition of this PR there are some patterns beginning to emerge around our stories that we might be able to abstract (i.e. main heading, description, subheadings/component permutations, etc). 

I think I'll file a ticket to create some sort of reusable interface for doing so...is there any reason you could see to _not_ do so at the moment?

Reply:
This is the direction we are starting to take with StoryHeading and StoryDescription. File that issue and tag Anne and Rachel. I would love to further standardize our stories.